recipes-core/base-files/base-files_%.bbappend

```diff
@@ -0,0 +1,10 @@
+do_install:append() {
```

Comment:
VPF is the Virtual Platform used to run Gen5 virtual device of Renesas. Currently, it is under
development so that it only allows us to use only 1 rootfs.ext4 file as rootfs.

Reply:
VPF stands for Virtual Platform which is used to run the Gen5 virtual device of Renesas. Currently, it is under development, which doesn't allow us to interact with the disk storage to create multiple partitions.

Reply:
I think that 1 partition is just a temporary solution while VPF is not fixed and/or gen5 chips are released. I would suggest moving alll the changes related to partition structure into a separate PR that will be marked as "DO NOT MERGE"

Reply:
Thanks for the explanation. Are there any "generic" changes that are needed just to enable gen5 and not for the VPF? Can we somehow isolate those?

Reply:
It is not urgent to isolate those. We can keep this pull request as "do not merge" and then create a clean one for gen5 as soon as we are close to getting the HW

Reply:
Ok, I got it.